Library/Formula/conserver.rb

```diff
@@ -1,13 +1,15 @@
-require 'formula'
-
 class Conserver < Formula
-  homepage 'http://conserver.com'
-  url 'http://conserver.com/conserver-8.1.19.tar.gz'
-  sha1 'e13974427a91f3740057fb1170378021f311e7ac'
+  homepage "http://conserver.com"
+  url "http://conserver.com/conserver-8.2.0.tar.gz"
+  sha256 "a05beb7b66c0228ece62786c3cfe4b0048c5d508e072fb3631c2bf460cab904f"
 
   def install
     system "./configure", "--prefix=#{prefix}"
     system "make"
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    system sbin/"conserver", "-h"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!